### DISCUSSION TITLE
Where can i find docs for urql/svelte v2?

### DISCUSSION THREAD

**Discussion by mpiorowski** [urql-graphql/urql#2494] [Q&A] [answered]
I was using operationStore everywhere, but now it doesn't work at all.

**Comment by freb** [comment]
The docs are not all the way up-to-date and mostly still reflect the pre v2 code. There is a small note at the top of the [Svelte binds](https://formidable.com/open-source/urql/docs/basics/svelte/) page that mentions this and says that the [API docs](https://formidable.com/open-source/urql/docs/api/svelte/) are up-to-date:

![image](https://user-images.githubusercontent.com/1723508/173475089-7898a425-1fbf-4009-992a-9565b7b79b1f.png)

There is an example of the new API in the examples directory, but it does not include a mutation or subscription example:

https://github.com/FormidableLabs/urql/blob/main/examples/with-svelte/src/PokemonList.svelte

I hope you will leave this as unanswered until the docs are updated.

**Comment by freb** [comment, marked as answer]
Looks like the new docs are coming soon and available here (thanks @JoviDeCroock):

https://formidable-com-urql-staging-2501.surge.sh/open-source/urql/docs/basics/svelte/
